test/unit/redirect.test.js

```diff
@@ -151,8 +157,12 @@ describe('redirect', () => {
         isSmartcarHosted: true,
         name: 'SmartcarAuthMessage',
         state,
+        make: null,
+        model: null,
+        vin: null,
+        year: null,
```

Comment:
we should have all the params defined in this test

Reply:
Fixed